client/src/pages/campaigns/filterPanel/FilterPanel.jsx

```diff
@@ -54,7 +54,7 @@ function FilterPanel({
           }}
           size={SIZE.compact}
           shape={SHAPE.pill}
-          isSelected={!(channel[0] || status[0])}
+          isSelected={!(channel[0] || status[0] || nameFilter[0])}
```

Comment:
"all" button is about channel and status filters only. its state doesn't depend on name filter.